core/templates/pages/topic-editor-page/topic-editor-page.component.ts

```diff
@@ -111,6 +111,19 @@ angular.module('oppia').directive('topicEditorPage', [
           ctrl.selectQuestionsTab = function() {
             TopicEditorRoutingService.navigateToQuestionsTab();
           };
+          ctrl.getNavbarText = function() {
+            if (TopicEditorStateService.hasLoadedTopic()) {
+              const activeTab = ctrl.getActiveTabName();
+              if (activeTab === 'main') {
+                return (
+                  `Topic Editor: ${ctrl.topic.getName()}
+                  (v${ctrl.topic.getVersion()})`);
+              }
+              if (activeTab.startsWith('subtopic')) {
```

Comment:
Why startsWith?

Reply:
Becuase we want this to be the same for subtopic_preview and subtopic_editor and hence startswith.

Reply:
Ah, I see. Thanks!